BDArmory/Radar/RadarUtils.cs

```diff
@@ -1388,6 +1389,146 @@ public static float GetStandoffJammingModifier(Vessel v, Competition.BDTeam team
             return modifiedSignature / signature;
         }
 
+        /*
```

Comment:
These commented out sections can be removed once you're satisfied with the unified `RadarTerrainNotchingCheck` function.